### PR TITLE
add sparse test, update existing 

### PR DESCRIPTION
**Pull request by raveenajain**
PR - https://github.com/highfidelity/hifi/pull/15555, https://github.com/highfidelity/hifi/pull/15827 (fix for expected images)

https://highfidelity.atlassian.net/projects/BUGZ/issues/BUGZ-278
https://highfidelity.manuscript.com/f/cases/22169/GLTF-improvements-sparse-accessors-vertex-colors-multiple-unnamed-materials-thin-bounding-boxes
https://highfidelity.atlassian.net/browse/BUGZ-598
https://highfidelity.atlassian.net/projects/BUGZ/issues/BUGZ-682

### CODE REVIEW

File: tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/lantern/test.js [hunk 1]
```diff
@@ -26,7 +26,7 @@ nitpick.perform("Read GLTF model", Script.resolvePath("."), "secondary", functio
         type: "Model",
         // https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Lantern
         modelURL: assetsRootPath + 'models/gltf_models/glb/Lantern.glb',
-        position: Vec3.sum(position, {x: 0.0, y: 0.2, z: -1.6 }),
+        position: Vec3.sum(position, {x: 0.0, y: 0.8, z: -2.75 }),
```
Comment (SamGondelman) [inline tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/lantern/test.js]: Why'd you make this one farther from the camera?

File: tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/oldCamera/test.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ nitpick.perform("Read GLTF model", Script.resolvePath("."), "secondary", functio
         type: "Model",
         // https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AntiqueCamera
         modelURL: assetsRootPath + 'models/gltf_models/glb/AntiqueCamera.glb',
-        position: Vec3.sum(position, {x: 0.0, y: 1.0, z: -0.8 }),
+        position: Vec3.sum(position, {x: 0.0, y: 1.0, z: -2.0 }),
```
Comment (SamGondelman) [inline tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/oldCamera/test.js]: This one too.

File: tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/lantern/test.js [hunk 3]
```diff
@@ -26,8 +26,8 @@ nitpick.perform("Read GLTF model", Script.resolvePath("."), "secondary", functio
         type: "Model",
         // https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Lantern
         modelURL: 'https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/Lantern/glTF/Lantern.gltf',
-        position: Vec3.sum(position, {x: 0.0, y: 0.2, z: -1.6 }),
-        rotation: Quat.fromPitchYawRollDegrees(0.0, -180.0, 0.0),
+        position: Vec3.sum(position, {x: 0.0, y: 0.8, z: -2.75 }),
+        rotation: Quat.fromPitchYawRollDegrees(0.0, 0.0, 0.0),
```
Comment (SamGondelman) [inline tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/lantern/test.js]: Why this change?

File: tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/oldCamera/test.js [hunk 4]
```diff
@@ -26,7 +26,7 @@ nitpick.perform("Read GLTF model", Script.resolvePath("."), "secondary", functio
         type: "Model",
         // https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AntiqueCamera
         modelURL: 'https://raw.githubusercontent.com/KhronosGroup/glTF-Sample-Models/master/2.0/AntiqueCamera/glTF/AntiqueCamera.gltf',
-        position: Vec3.sum(position, {x: 0.0, y: 1.0, z: -0.8 }),
+        position: Vec3.sum(position, {x: 0.0, y: 1.0, z: -2.0 }),
```
Comment (SamGondelman) [inline tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/oldCamera/test.js]: this one too

File: tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/lantern/test.js (hunk 1 above)
Comment (raveenajain) [reply tests/content/entity/model/modelReaders/gltfReader/glbTestSuite/lantern/test.js]: fixing the bounding boxes also fixed the positioning of a bunch of the models. instead of scaling down the model further to fit it entirely in frame, I pushed it back (I'll double check though; it's possible that it doesn't need to be pushed back as much to fit in frame)

File: tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/lantern/test.js (hunk 3 above)
Comment (raveenajain) [reply tests/content/entity/model/modelReaders/gltfReader/gltfTestSuite/lantern/test.js]: the rotation change isn't necessary; it was never updated because I was unable to render the correct image due to the previous LOD/culling issue
